src/elements/Step/StepGroup.js

```diff
@@ -48,6 +57,7 @@ StepGroup.propTypes = {
   /** Primary content of the StepGroup. Mutually exclusive with items prop. */
   children: customPropTypes.all([
     customPropTypes.mutuallyExclusive(['description', 'title']),
+    customPropTypes.ofComponentTypes(['Step']),    // <-- only allow Step children
```

Comment:
The inline comment can be removed here, this was just to point it out in the PR comment.